packages/components/src/Tabs/Tabs.tsx

```diff
@@ -131,22 +129,23 @@ export function InternalTab({
   },
 }: InternalTabProps) {
   const className = classnames(styles.tab, { [styles.selected]: selected });
-  const color = selected ? "green" : "heading";
 
   return (
-    <button
-      type="button"
-      role="tab"
-      id={label}
-      className={className}
-      onClick={event => {
-        activateTab();
-        onClick(event);
-      }}
-    >
-      <Typography element="span" size="base" textColor={color}>
-        {label}
-      </Typography>
-    </button>
+    <li role="presentation">
+      <button
+        type="button"
+        role="tab"
+        id={label}
+        className={className}
+        onClick={event => {
+          activateTab();
+          onClick(event);
+        }}
+      >
+        <Typography element="span" size="large" fontWeight="semiBold">
```

Comment:
I removed the `textColor` prop because it allows for `hover` and `focus` states to style it easily with CSS

Reply:
Nice.